api/docker-compose.yaml

```diff
@@ -115,6 +115,24 @@ services:
       command: wait-for db:3306 -- ./generate-cdr/init-new-cdr-db.sh --cdr-db-name public --run-list data --context local
       env_file:
         - db-cdr/vars.env
+  db-make-bq-denormalized-tables:
```

Comment:
Does this have to be a separate step? Can't we combine this with the other generate CDR BQ logic?